.github/workflows/publish.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       - name: Upload Release
         uses: xresloader/upload-to-github-release@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.WINGET_PUBLISH_PAT }}
```

Comment:
why would you change this token? doesn't the other workflow publish to winget?

Reply:
> When you use the repository's GITHUB_TOKEN to perform tasks, events triggered by the GITHUB_TOKEN will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs. For example, if a workflow run pushes code using the repository's GITHUB_TOKEN, a new workflow will not run even when the repository contains a workflow configured to run when push events occur.

If the release author is github-actions, it won't trigger any workflow runs and hence the publish-to-winget won't be triggered.